tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -106,6 +115,168 @@ Standard output:
 @fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/docs/src/prototypes/VirtualizedTree/VirtualizedTree.tsx
 @fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/docs/src/prototypes/VirtualizedTree/VirtualizedTree.tsx(64,31):
 @fluentui/docs: TS2538: Type 'undefined' cannot be used as an index type.
+@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/Avatar/Avatar.tsx
+@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/Avatar/Avatar.tsx(153,1):
+@fluentui/docs: TS2322: Type '{ name: PropTypes.Requireable<string>; image: (props: Record<string, any>, propName: string, componentName: string, ...args: any[]) => any; label: (props: Record<string, any>, propName: string, componentName: string, ...args: any[]) => any; ... 12 more ...; accessibility: PropTypes.Requireable<...>; }' is not assignable to type 'WeakValidationMap<WithAsProp<AvatarProps>>'.
+@fluentui/docs:   Types of property 'design' are incompatible.
+@fluentui/docs:     Type 'Requireable<InferProps<{ position: Requireable<string>; display: Requireable<string>; top: Requireable<string>; right: Requireable<string>; bottom: Requireable<...>; ... 16 more ...; maxHeight: Requireable<...>; }>>' is not assignable to type 'Validator<ComponentDesignProp>'.
+@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/Box/Box.tsx
+@fluentui/docs: ERROR in /office-ui-fabric-react/packages/fluentui/react-northstar/src/components/Box/Box.tsx(69,1):
+@fluentui/docs: TS2322: Type '{ styles: PropTypes.Requireable<object>; variables: PropTypes.Requireable<object>; design: PropTypes.Requireable<PropTypes.InferProps<{ position: PropTypes.Requireable<string>; display: PropTypes.Requireable<...>; ... 19 more ...; maxHeight: PropTypes.Requireable<...>; }>>; ... 5 more ...; accessibility: PropTypes.R...' is not assignable to type 'WeakValidationMap<WithAsProp<BoxProps>>'.
+@fluentui/docs:   The types of 'design[nominalTypeHack].position' are incompatible between these types.
+@fluentui/docs:     Type 'string' is not assignable to type 'PositionProperty'.
```

Comment:
this error is correct as far as I can tell; one CSS wrapper has a precise type and the other only has 'string'.

Reply:
it does disappear when building with dd6811fbe37f05a2b79427112a919d75a7417176 (from March 2), so office-ui-fabric will have to fix it somehow